### PR TITLE
[ACS-3548] Required ARIA attributes provided, buttons discernible text added, he…

### PR DESCRIPTION
**Pull request by jatin2008**
…ading label provided

**Please check if the PR fulfills these requirements**

> - [X ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ X] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
Required ARIA attributes were not provided, buttons were not having discernible text, headings label were not present.

**What is the new behaviour?**
Required ARIA attributes provided, buttons discernible text added, headings label added

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ X] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/content-services/src/lib/permission-manager/components/permission-list/permission-list.component.html [hunk 1]
```diff
@@ -25,6 +25,7 @@ <h3 class="adf-inherit-container-header">
 
                     <mat-slide-toggle
                         class="adf-inherit-toggle"
+                        aria-label="Toggle Permission"
```
Comment (DenysVuika) [inline lib/content-services/src/lib/permission-manager/components/permission-list/permission-list.component.html:28]: should be i18n resource key

**Review by DenysVuika — CHANGES_REQUESTED** [review]

File: lib/content-services/src/lib/permission-manager/components/permission-list/permission-list.component.html (hunk 1 above)
Comment (jatin2008) [reply lib/content-services/src/lib/permission-manager/components/permission-list/permission-list.component.html]: i18n resiuce key added

File: lib/core/src/lib/card-view/components/card-view-textitem/card-view-textitem.component.html [hunk 2]
```diff
@@ -12,6 +12,7 @@
             <input matInput
                    *ngIf="!property.multiline"
                    class="adf-property-value"
+                   title={{property.displayValue}}
```
Comment (DenysVuika) [inline lib/core/src/lib/card-view/components/card-view-textitem/card-view-textitem.component.html:15]: it should be the title of the property, not the value
```
title={{property.label}}
```

and for other inputs as well

**Review by DenysVuika — CHANGES_REQUESTED** [review]

File: lib/core/src/lib/card-view/components/card-view-textitem/card-view-textitem.component.html (hunk 2 above)
Comment (jatin2008) [reply lib/core/src/lib/card-view/components/card-view-textitem/card-view-textitem.component.html]: title="{{property.label | translate }}" has been added for all input fields.

**Comment by MichalKinas** [comment]
This PR doesn't contain fixes for issues provided in CSV file attached to the epic

File: lib/core/src/lib/card-view/components/card-view-textitem/card-view-textitem.component.html (hunk 2 above)
Comment (jatin2008) [reply lib/core/src/lib/card-view/components/card-view-textitem/card-view-textitem.component.html]: Resolved some accessibility issues present in CSV file and working on the remaining one's

Comment (jatin2008) [reply lib/core/src/lib/card-view/components/card-view-textitem/card-view-textitem.component.html]: Resolved some accessibility issues present in CSV file and working on the remaining one's

**Comment by jatin2008** [comment]
Issue id - 880237, 878585, 880256, 878352, 878353, 878354, 878359, 878739 fixed in CSV file for Accessibility issues, working on remaining one’s.

File: lib/content-services/src/lib/i18n/en.json [hunk 3]
```diff
@@ -425,7 +426,8 @@
       "DIRECT-PERMISSIONS": "Directly Applied Permission",
       "INHERITED-PERMISSIONS": "Inherited Permission",
       "INHERITED-SUBTITLE": "{{count}} users or groups are inheriting permission from a parent folder",
-      "SELECT-ROLE": "Select role"
+      "SELECT-ROLE": "Select role",
+      "INHERITED_PERMISSION_TOGGLE": "Inherited Permission Toggle"
```
Comment (DenysVuika) [inline lib/content-services/src/lib/i18n/en.json:430]:
```suggestion
      "INHERITED_PERMISSION_TOGGLE": "Toggle Inherited Permission"
```

**Review by DenysVuika — CHANGES_REQUESTED** [review]

File: lib/content-services/src/lib/i18n/en.json (hunk 3 above)
Comment (jatin2008) [reply lib/content-services/src/lib/i18n/en.json]: resolved requested change

**Review by DenysVuika — APPROVED** [review]

**Review by nikita-web-ua — APPROVED** [review]

File: lib/core/src/lib/comments/comments.component.html [hunk 4]
```diff
@@ -1,5 +1,5 @@
 <div class="adf-comments-container">
-    <div id="comment-header" class="adf-comments-header" role="heading">
+    <div id="comment-header" class="adf-comments-header" aria-level="3" role="heading">
```
Comment (AleksanderSklorz) [inline lib/core/src/lib/comments/comments.component.html:2]: In different PR I had discussion with Michał about that and seems like using h1/h2/h3 etc. elements is better than aria-level probably. Please read below links: 
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-level
"Opt for using the h1 thru h6 elements instead."

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/heading_role
"This defines the text in the div to be the main heading of the page, indicated by being level 1 via the aria-level attribute. Opt for using the h1 thru h6 element instead."

And also interesting explaining here, please read first answer:
https://webmasters.stackexchange.com/questions/137159/for-seo-is-role-heading-aria-level-1-the-same-as-h1

If you think it is better please change :)

Comment (jatin2008) [reply lib/core/src/lib/comments/comments.component.html:2]: i will check this @AleksanderSklorz :)

**Comment by MichalKinas** [comment]
What about 879018, 878715 (Aleksander fixed similar issue for snackbars, please check if this one is covered by that, if not apply similar solution)? Are those issues fixed?

**Comment by jatin2008** [comment]
Please refer https://github.com/Alfresco/alfresco-ng2-components/pull/7939 this pull request.

**Pull request closed without merging**